resync/api/monitoring_dashboard.py

```
@@ -754,62 +532,71 @@ def build_sample(rps: float) -> MetricSample:
+        try:
+            await lock.release()
+        except Exception as lock_error:
+            logger.debug("Lock release failed (possibly expired): %s", lock_error)
 
 
 async def metrics_collector_loop() -> None:
+    """Loop de coleta de métricas em background."""
     try:
         redis = get_redis_client()
         await redis.ping()
     except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)